# Hurley2022scRNA

Correct as at 24 March 2022

This repository contains all code used in manuscript:
Single-cell RNA Sequencing Demonstrates Inter-Patient Heterogeneity and Altered Tumor Microenvironment in Invasive and Intraductal Cribriform Prostate Cancer

# Organism

Homo sapiens
 	
# Overall design

scRNA-seq (whole transcriptome and TCR) performed on paired ICC/IDC-enriched and benign-enriched prostate samples obtained from radical prostatectomy.

# Brief Summary

Cribriform prostate cancer, found in both invasive cribriform carcinoma (ICC) and intraductal carcinoma (IDC), is an aggressive histological subtype that is associated with progression to lethal disease. To delineate the molecular and cellular underpinnings of ICC/IDC aggressiveness, this study examined paired ICC/IDC and benign prostate surgical samples by single-cell RNA sequencing, TCR sequencing and histology. ICC/IDC cancer cells expressed genes associated
with metastasis and targets with potential for therapeutic intervention. Pathway analyses and ligand/receptor status modeled cellular interactions among ICC/IDC and the tumor microenvironment (TME). The ICC/IDC TME was hallmarked by increased angiogenesis and immunosuppressive fibroblasts along with fewer T cells, increased T cell dysfunction, and elevated M2 polarization of macrophage subtypes. These findings support that cancer cell intrinsic pathways and a complex immunosuppressive TME contribute to the aggressive phenotype of ICC/IDC. These data highlight potential therapeutic opportunities to restore immune signaling in patients with ICC/IDC that may afford better outcomes.

# Software and Packages Information

Please see folder "220324_Software_Packages_Information" for comprehensive details on software and packages utilized in our data analysis, including:
- System requirements
- Installation guides
- Demo
- Instructions for use

# Requirement

you will need to download ngsperl to your local computer. Some python and R scripts from ngsperl were used in data analysis.

https://github.com/shengqh/ngsperl

you will need to install enclone for clonotype analysis

https://10xgenomics.github.io/enclone/
